waggle-dance-api/src/main/java/com/hotels/bdp/waggledance/api/model/ConnectionType.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (C) 2016-2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.bdp.waggledance.api.model;
+
+public enum ConnectionType {
+  DIRECT_CONNECTION,
```

Comment:
I think I'd prefer it if this was just called `DIRECT` as we already know it's related to a connection from the Enum name.

Reply:
sure will change.